crates/bevy_ecs/src/schedule/executor_parallel.rs

```diff
@@ -114,7 +111,13 @@ impl ParallelSystemExecutor for ParallelExecutor {
             self.events_sender = Some(sender);
         }
 
-        self.update_archetypes(systems, world);
+        for (index, container) in systems.iter_mut().enumerate() {
```

Comment:
can you add the tracing span back in here?

Reply:
is there any perf difference for updating the archetypes?

Reply:
Readded the tracing span and added a basic benchmark with a couple of cases, which does not show any difference for me.